serde_derive/src/ser.rs

```diff
@@ -898,6 +930,51 @@ fn wrap_serialize_with(
     })
 }
 
+fn wrap_serialize_seed(
+    value: Tokens,
+) -> Tokens {
+    quote!({
+        &_serde::ser::Seeded::new(__seed, #value)
+    })
+}
+
+
+fn wrap_serialize_seed_with(
```

Comment:
Largely copy pasted from `wrap_serialize_with` so this should be merged with that function.